src/pages/AdoptablePetsDetails.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PetDetail from "../components/PetDetail";
+import PetSidebar from "../components/PetSidebar";
+import { PetItems } from "../files/PetsData";
+
+const AdoptablePetsDetails = props => {
+  const { animalName = "Bubbles", animalId } = props.match.params;
+  const SelectedPet = PetItems.filter(item => item.animalId === animalId);
```

Comment:
sine this is a local variable, this really should be `selectedPet`. This will eventually be part of a hook that will fetch the data from a server.

Reply:
done